android/app/src/main/java/com/jellyfin/MainActivity.java

```diff
@@ -10,6 +10,6 @@
      */
     @Override
     protected String getMainComponentName() {
-        return "JellyFin";
+        return "jellyfin";
```

Comment:
Can you explain the naming convention?

Reply:
Its not a naming convention. I deleted app.json, and then recreated it, thats why the name is different. Do you want to use camel casing?

Reply:
Yes please 👍